### PR TITLE
1046 validate geometries in frontend

### PR DESCRIPTION
**Pull request by ahjyrkia**
closes https://github.com/HSLdevcom/jore-map-ui/issues/1046

### CODE REVIEW

**Comment by Divinare** [comment]
LGTM

**Comment by Divinare** [comment]
Can be reviewed after conflict fix

File: src/components/sidebar/nodeView/NodeView.tsx [hunk 1]
```diff
@@ -198,7 +228,6 @@ class NodeView extends ViewFormBase<INodeViewProps, INodeViewState> {
 
     private onNodeGeometryChange = (property: NodeLocationType, value: any) => {
         this.props.nodeStore!.updateNodeGeometry(property, value, NodeMeasurementType.Measured);
-        this.validateProperty(nodeValidationModel[property], property, value);
     };
 
     private onChangeNodeProperty = (property: keyof INode) => (value: any) => {
```
Comment (Divinare) [inline src/components/sidebar/nodeView/NodeView.tsx]: Hmm, maybe this line 206 can be removed and also maybe lines 207-209
Reason: now validation is done through watcher on each property change

Comment (ahjyrkia) [reply src/components/sidebar/nodeView/NodeView.tsx]: Removed lines 235-238. Validating seems to be working just fine without them

**Comment by Divinare** [comment]
Review ok

**Pull request merged**